### DISCUSSION TITLE
Layout development

### DISCUSSION THREAD

**Discussion by davidkdb** [QuestPDF/QuestPDF#801] [General]
Hi,

Is it possible to do the layout like this in QuestPDF?

![image](https://github.com/QuestPDF/QuestPDF/assets/57062216/12e6155d-6f06-46fb-bb67-eb42b524fcf3)

If so (especially the rounded boxes/pills), we would like to hire someone to help us with the details.

We have been using QuestPDF for quite a while, but this is something we have not done before.

**Comment by girlpunk** [comment]
The majority of the layout should be easily achievable using the standard row/column elements, or a table. Unfortunately the curved and dotted borders are not currently possible.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Since this discussion was opened, our documentation has undergone a significant rewrite with expanded examples and use cases. We are also considering adding a layout similar to yours as a documented use case to help other users achieve comparable results.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
